packages/strapi-design-system/src/ModalLayout/__tests__/ModalLayout.spec.js

```diff
@@ -462,9 +462,7 @@ describe('ModalLayout', () => {
         max-height: 60vh;
       }
 
-      <body
-        class="lock-body-scroll"
-      >
+      <body>
```

Comment:
I'm wondering about this change. According to `useLockScroll()` the class should still be there, don't you think?

Reply:
no you're right, I didn't realise `useLockScroll` required an argument whether to do it or not. I've set this to true because Modals must be removed from the tree unlike `Dialog`.